cloudify_agent/resources/pm/nssm/nssm.conf.template

```diff
@@ -16,10 +16,13 @@ if %errorlevel% neq 0 exit /b %errorlevel%
 echo Setting service environment
 
 "{{ nssm_path }}" set {{ name }} AppEnvironmentExtra ^
-REST_HOST={{ rest_host }} ^
+REST_HOST={{ rest_host|join(',') }} ^
 REST_PORT={{ rest_port }} ^
 LOCAL_REST_CERT_FILE="{{ local_rest_cert_file }}" ^
-MANAGER_FILE_SERVER_URL=https://{{ rest_host }}:{{ rest_port }}/resources ^
+MANAGER_FILE_SERVER_URL={% for host in rest_host -%}
+    https://{{ host }}:{{ rest_port }}/resources
+    {%- if not loop.last %},{% endif %}
+{%- endfor %} ^
```

Comment:
can't do `|map|join` because jinja doesn't really support map+format. This loop is written in multiple lines, and whitespace is controlled by the `{%-`